addons/mail/static/src/models/composer/composer.js

```diff
@@ -620,6 +628,16 @@ function factory(dependencies) {
             compute: '_computeMentionedPartners',
             dependencies: ['textInputContent'],
         }),
+        threadSuggestedPartners: many2many('mail.suggested_partner', {
+            related: 'thread.suggestedPartners'
+        }),
+        recipients: attr({
+            compute: '_getRecipients',
+            dependencies: [
+                'thread',
+                'threadSuggestedPartners',
+            ]
+        }),
```

Comment:
missing descriptive comment. Suggested format:
- (mandatory) 1st paragraph: give the _"what"_
- (optional) 2nd paragraph: give the _"why"_
- (optional) 3rd paragraph: give the "extra info"

Here I expect to see _"what"_ and _"why"_.